screenpipe-server/src/server.rs

```diff
@@ -501,9 +462,6 @@ impl Server {
             .route("/audio/stop", post(stop_device))
             .route("/audio/status", post(get_device_status))
             .route("/audio/list", get(get_devices))
```

Comment:
we could remove audio restart api too, just dead code atm until we ensure it work well